### PR TITLE
MBS-9909: Change reports to use React (8 - ISRCs)

### PR DESCRIPTION
**Pull request by reosarevok**
Also changed ISRC to Isrc in the same way we did with ISWCs.

### CODE REVIEW

File: root/report/IsrcsWithManyRecordings.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+              <th>{l('Length')}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {items.map((item, index) => {
+              lastIsrc = currentIsrc;
+              currentIsrc = item.isrc;
+
+              return (
+                <>
```
Comment (yvanzo) [inline root/report/IsrcsWithManyRecordings.js]: Should be `<React.Fragment key={…}>` here. Found the same issue with already converted `CollaborationRelationships`, `DuplicateArtists`, and `IswcsWithManyWorks`.

Comment (reosarevok) [reply root/report/IsrcsWithManyRecordings.js]: Done (I think?)

Comment (yvanzo) [reply root/report/IsrcsWithManyRecordings.js]: Almost, except that key should probably be a concatenation of children’s keys rather than `index`.

Comment (reosarevok) [reply root/report/IsrcsWithManyRecordings.js]: Better?

Comment (yvanzo) [reply root/report/IsrcsWithManyRecordings.js]: Yep, thank you! I just edited your commit to restore `key` attributes on `tr` elements.

Comment (reosarevok) [reply root/report/IsrcsWithManyRecordings.js]: Aren't the keys only supposed to be on the top element of the map? That's what I understood from @mwiencek, that those tr keys were unneeded.

Comment (yvanzo) [reply root/report/IsrcsWithManyRecordings.js]: Maybe I have been a bit hasty indeed, but keys don’t seem to hurt.

File: root/report/IsrcsWithManyRecordings.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+              lastIsrc = currentIsrc;
+              currentIsrc = item.isrc;
+
+              return (
+                <React.Fragment key={item.isrc + '-' + item.recording.gid}>
+                  {lastIsrc === item.isrc ? null : (
+                    <tr className="even" key={item.isrc}>
+                      <td>
+                        <a href={'/isrc/' + item.isrc}>{item.isrc}</a>
+                        <span>{' (' + item.recordingcount + ')'}</span>
```
Comment (mwiencek) [inline root/report/IsrcsWithManyRecordings.js]: `bracketed`?

**Comment by reosarevok** [comment]
Added a few eslint fixes and changed old URL files to Url too in the reports folder.

File: root/report/IsrcsWithManyRecordings.js [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+              lastIsrc = currentIsrc;
+              currentIsrc = item.isrc;
+
+              return (
+                <React.Fragment key={item.isrc + '-' + item.recording.gid}>
+                  {lastIsrc === item.isrc ? null : (
+                    <tr className="even">
+                      <td>
+                        <a href={'/isrc/' + item.isrc}>{item.isrc}</a>
+                        <span>{bracketed(item.recordingcount)}</span>
```
Comment (mwiencek) [inline root/report/IsrcsWithManyRecordings.js]: Pretty sure you need a `{' '}` between the a and span too since bracketed doesn't include one (sorry)

Comment (reosarevok) [reply root/report/IsrcsWithManyRecordings.js]: Done (also for ISWCs)

**Pull request merged**